README.md

```diff
@@ -1 +1,167 @@
-# witnet-block-relay
\ No newline at end of file
+# witnet-ethereum-block-relay
+
+`witnet-ethereum-block-relay` is an open source implementation of the Block Relay in Witnet.
+
+ DISCLAIMER: this is a work in progress, meaning the contract could be voulnerable to attacks.
```

Comment:
this is a work in progress, by which we mean the contract could still be vulnerable to attack. Use at your own risk.